src/resources/views/crud/fields/relationship/repeatable_relation.blade.php

```diff
@@ -0,0 +1,33 @@
+@php
```

Comment:
Could you please add a comment block above @pxpm to explain why this is used and when?